### ISSUE TITLE
Highmaps interactivity gone.

### ISSUE THREAD

**Issue by Shuyinsama** [alvarotrigo/fullPage.js#2004]
Hello,

I am using fullpage.js in combination with some charts that I create using the highcharts library.
I've been trying a couple of things now but I can't seem to figure this out.

I use the Highmaps to create a interactive map of my country but inside fullpage.js the hover events for the chart don't work.

Is this a know issue? Am I doing something wrong?

The slide you can find here: http://development.redactiepartners.nl/dev/vanlanschot/#hoofdstuk1_kaart
It's small for now but the blue parts in the map should be hoverable and have a tooltip.

**Comment by alvarotrigo** [comment]
Make sure you initialize it in the `afterRender` callback as detailed in the [fullpage.js FAQs](https://github.com/alvarotrigo/fullPage.js/wiki/FAQ---Frequently-Answered-Questions).

If it still not working please provide an isolated reproduction in jsfiddle or codepen without any external CSS or JS (unless required) as detailed in the [reporting issues guidelines](https://github.com/alvarotrigo/fullPage.js#reporting-issues)

**Comment by Shuyinsama** [comment]
My apologies I don't need to make issues when I'm in a hurry.

I reproduced it on [JSFiddle](http://jsfiddle.net/yg7cz8m9/).
It's on the second section. In my standalone "test" the hovers work fine of course but here they don't even if put inside the afterRender method.

I might need to use jQuery delegation. I also tried looking if it was a z-index issue but as far as I can tell it's not that.

Thanks for looking!
